apps/web/src/app/(networks)/(non-evm)/stellar/_common/lib/soroban/dex-factory-helpers.ts

```diff
@@ -118,7 +118,7 @@ export async function createAndInitializePool({
     console.log('⏳ Waiting for confirmation...')
 
     // Wait for confirmation
-    const txResult = await waitForTransaction(submitResult.hash)
+    const txResult = await waitForTransaction(submitResult.hash, 60000, 2)
```

Comment:
This extends the normal timeout for this and waits for an extra ledger confirmation and replaces the downstream hard-coded delay of 8 seconds used by the pool add page.

Having the delay be built into this function makes things easier to reason about by guaranteeing that when this promise resolves successfully, we actually can assume the pool is created and initialized.

This eliminates some jank with the hard-coded 8 second delay being outside the mutation hook that calls this. Namely, the hook was spawning the success toast before the delay is done and communicating "we are done creating the pool but are waiting a bit before adding liquidity" is confusing. Now, we only consider the pool creation done (and start adding liquidity) once we are confident the pool creation is ready on-chain.